### PR TITLE
Add day of week to `user-local-time`

### LINKED ISSUE #4309: Indicate yesterday/tomorrow in `user-local-time`
Would it be possible to add something like `(Yesterday)` or `(Tomorrow)` if the user's local time is a different day than my own?

Btw, `user-local-time` is definitely one of my _favorite_ rgh features!! ❤️

### PR DESCRIPTION
**Pull request by yakov116**
Closes #4309 

## Test URLs

My Icon

## Screenshot

image

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]
Really simple change.  I wouldn't have guessed 😄

**Comment by fregante** [comment]
I'd rather only show it if it's a different day to improve the signal-to-noise.

**Comment by yakov116** [comment]
> I'd rather only show it if it's a different day to improve the signal-to-noise.

Was thinking about that too. Worth the extra code?

Also should the day be at the end or like it is now is fine?

**Comment by bl-ue** [comment]
Are we going to [remove all but the timestamp from the tooltip](https://github.com/sindresorhus/refined-github/issues/4309#issuecomment-830795100)?  I, for one, would definitely prefer it to the current, longer version.

**Comment by yakov116** [comment]
I like that information, it tells me from what commit and how long ago the person publicly committed.

File: source/features/user-local-time.tsx [hunk 1]
```diff
@@ -135,7 +139,8 @@ function init(): void {
 
 		const now = new Date();
 		now.setMinutes(parseOffset(date) + now.getTimezoneOffset() + now.getMinutes());
-		placeholder.textContent = timeFormatter.format(now);
+		const userDayOfWeek = dayOfWeek.format(new Date(now));
+		placeholder.textContent = timeFormatter.format(now) + (userDayOfWeek === dayOfWeek.format(new Date()) ? '' : ' ' + userDayOfWeek);
```
[the remark this answers is not in the document]
Comment (yakov116) [reply source/features/user-local-time.tsx:143]: Duhh guess it was after my bedtime.

**Review by fregante — APPROVED** [review]
Untested but it looks ok, even if the date ended up being before the time

**Comment by bl-ue** [comment]
🎉 

When's this going to be available?  Or rather, what is your release schedule (if any)?

**Comment by yakov116** [comment]
> 🎉
> 
> When's this going to be available? Or rather, what is your release schedule (if any)?

Unless something big comes up probably next week sometime.

**Comment by bl-ue** [comment]
Sounds good.  Thank you!

**Pull request merged**